src/main/java/org/socialculture/platform/member/service/MemberServiceImpl.java

```diff
@@ -291,4 +290,20 @@ public void changeFirstLogin(String email) {
         member.markFirstLoginComplete();
         memberRepository.save(member);
     }
+
+    /**
+     * 사용자의 이메일을 가지고 memberId를 조회
+     *
+     * @param email
+     * @return
+     */
+    @Override
```

Comment:
email로 memberId를 조회하는 코드를 service에 구현하셨군요!
궁금한 점이 생겨서 여쭤봅니다.
이렇게 비즈니스 로직없이 단순히 email로 memberId를 조회하는 코드는 레포지토리에서 @Query("SELECT m.memberId FROM MemberEntity m WHERE m.email = :email")
```suggestion
    @Override
```
Optional<Long> findMemberIdByEmail(@Param("email") String email); 이렇게 정의해서
사용하는 방법은 어떻게 생각하시는지, 서비스단에 구현한 이유가 있으실지 궁금합니다!

Reply:
JPQL로 안하실려고 하신 것 같네요?